### ISSUE TITLE
Updating Meta learner parameters

### ISSUE THREAD

**Issue by smkh1985** [gabrielhuang/reptile-pytorch#8]
It's mentioned in the original article that update's rule is : Phi <--- Phi - Eps * (W - Phi)
why didn't you use Eps and why you used (Phi-W)

**Comment by gabrielhuang** [comment]
The learning rate can be tuned through the optimizer, and the minus sign is
due to the fact that the optimizer goes in the opposite direction to the
gradient (therefore the sign is flipped)

**Comment by Minhvt34** [comment]
Hi author,
I am curious about the "eps" parameter. To perform SGD for the meta step, why don't you use eps?
